android/src/main/java/com/applovin/reactnative/AppLovinMAXAdViewManager.java

```diff
@@ -41,6 +45,26 @@ public Map<String, Object> getExportedCustomDirectEventTypeConstants()
                 .build();
     }
 
+    @Nullable
+    @Override
+    public Map<String, Integer> getCommandsMap()
+    {
+        return MapBuilder.of(
+                "loadAd", COMMAND_LOAD_AD
```

Comment:
Where is `COMMAND_LOAD_AD` or its value of `1` actually used?

Reply:
`COMMAND_LOAD_AD` is used in `receiveCommand()`, which is defined just under `getCommandsMap()`.

Reply:
Yeah - was wondering if there's a more direct way of receiving/identifying the command (e.g. via `"loadAd"` string directly like on Flutter).